src/Messenger/IndexationRequestSpoolSubscriber.php

```diff
@@ -16,15 +16,13 @@
  */
 class IndexationRequestSpoolSubscriber implements EventSubscriberInterface, ResetInterface
 {
-    private $logger;
-
     private $wasExceptionThrown = false;
 
     private $singleTransport;
 
     private $bus;
 
-    public function __construct(TransportInterface $singleTransport, MessageBusInterface $bus, LoggerInterface $logger = null)
+    public function __construct(TransportInterface $singleTransport, MessageBusInterface $bus)
     {
         $this->logger = $logger;
```

Comment:
This assignation should also be removed I guess?

Reply:
oups thanks, fixed